### PR TITLE
Create LIT6809MEMiracleSickMonk.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Created to entry for Miracle no. 7 following the example of LIT6808MEMiracleChurch

### CODE REVIEW

File: new/LIT6809MEMiracleSickMonk.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6809MEMiracleSickMonk" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of ʾEwosṭatewos on a sick monk</title>
```
Comment (eu-genia) [inline new/LIT6809MEMiracleSickMonk.xml:8]: lang="en"

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6809MEMiracleSickMonk.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6809MEMiracleSickMonk.xml:8]: I would also extend the title to make it easier identifiable (a sick monk from Dabra Māryām and ʾAbuna ʾAbsādi)

Comment (CarstenHoffmannMarburg) [reply new/LIT6809MEMiracleSickMonk.xml:8]: Done.

**Review by eu-genia — APPROVED** [review]
perfect, thank you!

**Pull request merged**